compiler/rustc_span/src/source_map/tests.rs

```diff
@@ -567,3 +567,30 @@ fn test_next_point() {
     assert_eq!(span.hi().0, 6);
     assert!(sm.span_to_snippet(span).is_err());
 }
+
+#[cfg(unix)]
```

Comment:
this should be `#[cfg(linux)]` since those files aren't available on some other systems, e.g. I'd be surprised it they were on Apple or BSDs.